src/components/molecules/TableView.tsx

```diff
@@ -29,7 +29,7 @@ const StyledTableCell = withStyles((theme: Theme) =>
     },
     sizeSmall: {
       padding: '5px 0px',
-      border: '1px solid rgba(0, 0, 0, 0.12)',
+      border: `1px solid ${sizeSmallBorderColor}`,
```

Comment:
The point of making colors global is to make colors reuseable.
rename`sizeSmallBorderColor` to `borderColor`